core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -66,6 +66,12 @@ class TaskInfo(
    */
   var finishTime: Long = 0
 
+  /**
+   * Set this tag when this task killed by other attempt. This kind of task should not resubmit
+   * while executor lost.
+   */
```

Comment:
nit: calling it a "tag" doesn't aid understanding, but rather leaves me wondering why it was called a "tag" instead of a "var" or something more descriptive.

more consequential: Please explain what defines "this kind" of task.

Reply:
Thanks for reviewing, rewrite this comment.